src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -6086,20 +6123,82 @@ class D6 : D5
     [Obsolete]
     public override void goo() {}
 }
+
+class E1
+{
+    public virtual int Goo {get; set;}
+}
+class E2 : E1
+{
+    public override int Goo { [Obsolete] get; set;}
```

Comment:
>[Obsolete] [](start = 30, length = 10)

Please make sure we have a separate unit-test for consumption of an overridden property, i.e. the Obsolete attribute is on different levels of inheritance hierarchy, including cases when not all accessors are overridden in a derived class

Reply:
We completely ignore any obsolete attributes on an overridden member, and only look at the attributes on the base member. This is true for accessors, properties, methods, etc.